app/uk/gov/hmrc/overseaspensiontransferbackend/connectors/SubmissionConnector.scala

```diff
@@ -75,12 +83,64 @@ class SubmissionConnectorImpl @Inject() (
       .execute[HttpResponse]
       .map(handleDownstreamResponse)
   }
+
+  override def getAllSubmissions(pstrNumber: PstrNumber)(implicit hs: HeaderCarrier): Future[Either[DownstreamGetAllError, DownstreamGetAllSuccess]] = ???
 }
 
 @Singleton
 class DummySubmissionConnectorImpl @Inject() ()(implicit ec: ExecutionContext) extends SubmissionConnector {
 
-  override def submit(validated: ValidatedSubmission)(implicit hc: HeaderCarrier): Future[Either[DownstreamError, DownstreamSuccess]] = {
-    Future.successful(Right(DownstreamSuccess(QtNumber("QT123456"), Instant.now(), "formBundleNumber")))
+  override def submit(validated: ValidatedSubmission)(implicit hc: HeaderCarrier): Future[Either[DownstreamSubmittedError, DownstreamSubmittedSuccess]] = {
+    Future.successful(Right(DownstreamSubmittedSuccess(QtNumber("QT123456"), Instant.now(), "formBundleNumber")))
+  }
+
+  override def getAllSubmissions(pstrNumber: PstrNumber)(implicit hs: HeaderCarrier): Future[Either[DownstreamGetAllError, DownstreamGetAllSuccess]] = {
+
+    val payload = DownstreamGetAllSuccess(
```

Comment:
I think that all of this should be coming from stubs, instead of hard coded in the backend. What if we forget to remove some or all of this. We need to be merging shippable, prod-ready stuff. This implementation should make the request to ETMP (but be overridden by config to talk to our stub locally)